src/packages/generate-license-file/src/lib/utils/pnpmCli.utils.ts

```diff
@@ -0,0 +1,38 @@
+﻿import { promisify } from "util";
+import { exec } from "child_process";
+
+const execAsync = promisify(exec);
+
+export type PnpmVersion = {
+  major: number;
+  minor: number;
+  patch: number;
+};
+
+type PnpmLsJsonStdOut = {
+  [spdxKey: string]: Dependency[];
+};
+
+export type Dependency = {
+  name: string;
+  version: string;
+  path: string;
+};
+
+export const getPnpmVersion = async (): Promise<PnpmVersion> => {
+  const { stdout } = await execAsync("pnpm --version");
+
+  const [major, minor, patch] = stdout.split(".").map(v => +v);
```

Comment:
`.map(v => +v)` is snazzy but I don't think what it's doing is clear/_immediately_ obvious 😛 (it sits in the `!!` bucket for me, aha).

Would you oppose this being more explicit ie `.map(Number)`?

Reply:
Yeah `+v` is defo in the same category as `!!` for me. Very happy to change to `.map(Number)`